### PR TITLE
chore(root): upgraded to babel 7 and fixed existing compilation errors

### PR DESCRIPTION
**Pull request by DalderupMaurice**
- Upgraded to babel 7 
- Updated tests snapshots
- Fixed enzym tests issue
- Fixed compilation warnings
- Moved a dep from dependencies to devDependencies (which belongs there)

### CODE REVIEW

File: src/app/components/Login/Login.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export class Login extends Component {
         // getTransactions(networks, selectedNetworkId, account).then(transactions => setTransactions(transactions))
         Neoscan.setNet(selectedNetworkId)
         Neoscan.getBalance(account.address).then(results => setBalance(results.neo, results.gas))
-        Neoscan.getTxsByAddress(account.address).then(results => setTransactions(results))
+        Neoscan.getTxsByAddressUrl(account.address).then(results => setTransactions(results))
```
Comment (phetter) [inline src/app/components/Login/Login.js]: I think this is a bug.

Comment (DalderupMaurice) [reply src/app/components/Login/Login.js]: The method doesn't exist in the Neoscan file :P 
Also got a compilation error pointing this out. I also did a search and wasn't able to find the function

Comment (phetter) [reply src/app/components/Login/Login.js]: Weird. I'll check into it.

Comment (DalderupMaurice) [reply src/app/components/Login/Login.js]: keep me posted! 🎉

Comment (DalderupMaurice) [reply src/app/components/Login/Login.js]: Alright. So I can remove this line and then we're done for this PR?

**Pull request merged**